projects/carba-detect/sawtooth.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+#-------------------------------------------------------------------------------
+# Name:        MCP4921_sawtooth.py
+# Purpose:     Output a sawtooth waveform
+#-------------------------------------------------------------------------------
+
+import spidev
+from time import sleep
+import math
+
+spi_max_speed = 4 * 1000000 # 4 MHz
+V_ref = 5 # 5V 
+max_value = (2**12)-1 # 12 bits for the MCP 4921
+CE = 1 # CE0 or CE1, select SPI device on bus
+
+# setup and open an SPI channel
+spi = spidev.SpiDev()
+spi.open(0,CE)
+spi.max_speed_hz = spi_max_speed
+
+freq = 90
```

Comment:
What does this frequency mean?
Does it make sense to pass it as a command line parameter?